crypto/threads_lock_contention.c

```diff
@@ -0,0 +1,245 @@
+/*
```

Comment:
I think this won't build on non-posix platforms (windows). If you want go that path, then either the whole file needs infdef _WIN guard or better the Configure script should not let this file in when build is being prepared for windows.